game.c

```diff
@@ -677,7 +677,7 @@ void main()
 			exit(0);
 		}
 
-		msg(1,"21 Chocosticks loaded on the Tray",2,0);
+		msg(1,"twenty one Chocosticks loaded on the Tray",2,0);
```

Comment:
While the other 2 changes below are ok. This one looks like it make it worse (i know! subjective :smile: )

I am OK to merge the other two changed lines if you really want.
Could you please update this Pull Request?